model/src/main/kotlin/CuratedPackage.kt

```diff
@@ -20,7 +20,7 @@
 package org.ossreviewtoolkit.model
 
 /**
- * A [Package] including the [PackageCurationResult]s that were applied to it, in order to be able to trace back how the
+ * A [Package] including the [PackageCuration]s that were applied to it, in order to be able to trace back how the
```

Comment:
Basically, the PR looks sensible to me. However, the `PackageCurationResult` class should be removed completely now as well.

Reply:
Commit message:

- "results contain*d* entries"
- "in particular the value~s~"
- "original value~s~"
- Comma before "had been useful"
- "ORT result ha~s~ve been refactored"
- To avoid multiple "So, " sentences:
  - "So, that the serialized ORT result" -> "This means that the serialized ORT result"
  - "So, the `base` property is not needed anymore." -> "That is why ..."
- "Remove the `base` property, to also remove the `diff()` function, which
is only needed for computing the `base`." -> "So, remove the `base` property, which allows to also remove the `diff()` function as that was only needed to compute the `base`."
- I'd just drop "In particular when `Map`s are in play.".
- "It may may" -> "It may make"
- Do not put "property" into backticks.

Reply:
"...including the [PackageCurationData] that was applied to it..."

Reply:
Not required for this PR, but looking at this class now I wonder if we should also change the `metadata` property of this class to contain the raw metadata of the package and add a helper function to get the package with applied curations instead.

Reply:
> Not required for this PR,
I thought about this too. (Also thought it could be a follow up), had basically similar idea in mind.